### PR TITLE
tickets/DM-32085: Implemented the configuration registry

### PR DESCRIPTION
**Pull request by rbovill**
Added additional script configurations.

### CODE REVIEW

File: python/lsst/ts/IntegrationTests/config_registry.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+#
+# You should have received a copy of the GNU General Public License
+
+from collections import UserDict
+
+__all__ = ["registry"]
+
+
+class ConfigRegistry(UserDict):
+    """Add the given named configuration to the registry dictionary.
```
Comment (mareuter) [inline python/lsst/ts/IntegrationTests/config_registry.py:27]: The documentation is in the wrong place. This should be in the `__setitem__` function not at the class level.

File: python/lsst/ts/IntegrationTests/take_image_latiss_configs.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+import yaml
+from .config_registry import registry
+
+
+# Add the script configurations to the configuration registry.
+
+registry["auxtel_visit_config1"] = yaml.safe_dump(
+    {
+        "nimages": 1.0,
```
Comment (mareuter) [inline python/lsst/ts/IntegrationTests/take_image_latiss_configs.py:30]: All nimages should be integers. Hard to take a decimal number of exposures ... unless the camera breaks during one. :-)

File: python/lsst/ts/IntegrationTests/base_script.py [hunk 3]
```diff
@@ -38,10 +38,6 @@ class BaseScript:
 
     Parameters
     ----------
-    config : `str`
```
Comment (mareuter) [inline python/lsst/ts/IntegrationTests/base_script.py:41]: Didn't catch this before, but the Parameters section here belongs to the `__init__` function.

**Review by mareuter — APPROVED** [review]
Some changes to make before merging, but looks good.

File: python/lsst/ts/IntegrationTests/take_image_latiss_configs.py (hunk 2 above)
Comment (rbovill) [reply python/lsst/ts/IntegrationTests/take_image_latiss_configs.py]: Fixed

File: python/lsst/ts/IntegrationTests/config_registry.py (hunk 1 above)
Comment (rbovill) [reply python/lsst/ts/IntegrationTests/config_registry.py]: Rearranged as requested.

**Pull request merged**